applications/ExternalSolversApplication/external_includes/feast_solver.h

```diff
@@ -174,6 +174,7 @@ class FEASTSolver: public LinearSolver<TSparseSpaceType, TDenseSpaceType,
             "solve_eigenvalue_problem": true,
```

Comment:
@msandre I added echo_level and changed the CammelCase

Reply:
looks good, thanks!